environment.yml

```diff
@@ -6,12 +6,12 @@ dependencies:
 - pyyaml
 - pip
 - ipython
-- toml
+- flit-core
```

Comment:
That’s a bit weird: When I run beni, that’s the difference in the environment.yml it creates. Why?

Reply:
Well, if we enable the checks to run on PRs, that can’t happen anymore because the file is now used in the tests.

Reply:
Yep that would be good!